docroot/modules/custom/mass_content/mass_content.install

```diff
@@ -1009,3 +1009,19 @@ function mass_content_update_8016(&$sandbox) {
     }
   }
 }
+
+/**
+ * Delete field_bg_narrow from Organization content type.
```

Comment:
🤔 is this actually needed? On my local, if I disable this update hook, config import still works to delete the field.

It's not actually _breaking_ anything to have this, this is more a curiosity on my part.

Reply:
Right, removed the update hook